src/news/_newswidget.py

```
@@ -55,27 +50,22 @@ def __init__(self, *args, **kwargs) -> None:
         name = os.path.basename(img_url)
         self._downloader.download(name, self._images_dl_request, img_url)
 
-    def add_image_resource(self, image_name: str, image_path: str) -> None:
-        doc = self.newsTextBrowser.document()
-        if doc.resource(QTextDocument.ResourceType.ImageResource, QUrl(image_name)):
-            return
-        img = QImage(image_path)
-        scaled = img.scaled(QSize(600, 338))
-        doc.addResource(QTextDocument.ResourceType.ImageResource, QUrl(image_name), scaled)
-
     def item_image_downloaded(self, image_name: str, result: tuple[str, bool]) -> None:
```

Ruff (0.14.13)

Unused method argument: `image_name`

(ARG002)